src/components/samples/SamplesListContent.js

```diff
@@ -93,10 +98,20 @@ const SamplesListContent = ({
       width: 85,
     }
   ];
-
+  const listExport = () =>
+    withToken(token, api.samples.listExport)().then(res => res.data)
 
   return <>
-    <AppPageHeader title="Samples & Extractions" extra={actionsToButtonList("/samples", actions)} />
+    <AppPageHeader title="Samples & Extractions"
+                   extra={[
+                     <ExportButton
+                       exportFunction={listExport}
+                       fileName={'samples'}
+                     />
+                     ,
+                     ...actionsToButtonList("/samples", actions)
+                   ]}
+    />
```

Comment:
Incorrect indentation, see the rest of the project and use a similar indentation style.